src/test/java/htsjdk/variant/vcf/VCFEncoderTest.java

```diff
@@ -90,6 +90,11 @@ public void testVCFWriterDoubleFormatTestData(final double d, final String expec
         tests.add(new Object[]{keepMissing, vc, "./.:.:2", alleleMap, formatKeys});
         baseGT.noAttributes();
 
+        //check that we only produce a single . when writing attributes with multiple values instead of .,.
```

Comment:
please add whitespace before text in `//` comments
```suggestion
        // check that we only produce a single . when writing attributes with multiple values instead of .,.
```